tests/test_matrix_utils_connect.py

```
@@ -0,0 +1,187 @@
+    """Missing matrix_rooms should raise ValueError."""
+    config = {
+        "matrix": {
+            "homeserver": "https://example.org",
+            "access_token": "token",
+            "bot_user_id": "@bot:example.org",
+        }
+    }
+
+    with (
+        patch("mmrelay.matrix_utils.os.path.exists", return_value=False),
+        pytest.raises(ValueError),
```

Ruff (0.15.9)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)